src/gui/SubWindow.cpp

```diff
@@ -111,12 +95,17 @@ SubWindow::SubWindow(QWidget *parent, Qt::WindowFlags windowFlags) :
  */
 void SubWindow::paintEvent( QPaintEvent * )
 {
+	// Don't paint any of the other stuff if the sub window is maximized
+	// so that only its child content is painted.
+	if (isMaximized())
```

Comment:
Would it make sense to inline this block?

Reply:
I mean put the return statement in the same line as the if, with braces.

Reply:
Ah, ok. Done with commit d5be2e5e6ad. For some reasons I had the code of `adjustTitleBar` in mind when reading the question. That method has a bit more code in the scope of the `if (isMaximized())` statement.